src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/WindowsRuntime/Windows/UI/ViewManagement/InputPaneRcw.cs

```diff
@@ -19,6 +19,36 @@ namespace Windows.UI.ViewManagement
         /// </summary>
         internal static class InputPaneRcw
         {
+            [DllImport("api-ms-win-core-winrt-string-l1-1-0.dll", CallingConvention = CallingConvention.StdCall)]
+            private static extern unsafe int WindowsCreateString([MarshalAs(UnmanagedType.LPWStr)] string sourceString,
+                                                  int length,
+                                                  out IntPtr hstring);
+
+            [DllImport("api-ms-win-core-winrt-string-l1-1-0.dll", CallingConvention = CallingConvention.StdCall)]
```

Comment:
Assembly names used in DllImport attributes should be defined as costants in a common file. 

https://github.com/dotnet/wpf/blob/ac9d1b7a6b0ee7c44fd2875a1174b820b3940619/src/Microsoft.DotNet.Wpf/src/Shared/RefAssemblyAttrs.cs